### DISCUSSION TITLE
Send Captcha to Discord Server

### DISCUSSION THREAD

**Discussion by brycewiedemeier** [jef/streetmerchant#2872] [Help]
I would like to send the captcha to a discord server instead of have it directly dm me so that other people can respond to the captchas. If somebody could help me modify the code that would be sick because I don't know a thing about javascript. Any help appreciated!

**Comment by Mr-Tech-13** [comment]
I did a bit of tinkering, and you could send the captcha to a channel, but others couldnt respond to it. I think it has to do with how the message is sent.
